src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/on.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'on(event, handler)'
+excerpt: 'Browser module: page.on method'
+---
+
+Allows the definition of an event handler function that will be executed every time the specified event is emitted.
+
+| Parameter       | Type   | Default | Description                                                                                                                                                                                                                           |
+|-----------------|--------|---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| event        | string  | `''`    |  Event to attach handler to. Currently, only `'console'` event is supported.                                                                                                               |
+| handler        | function  | `null`    |  A function to be called every time the specified event is emitted.                                                                                                               |
+
+
+<Blockquote mod="warning" title="">
+
+When using `page.on` method, the page has to be explicitly [closed](/javascript-api/k6-experimental/browser/page/close/) for the iteration to be able to finish.
```

Comment:
```suggestion
When using the `page.on` method, the page has to be explicitly [closed](/javascript-api/k6-experimental/browser/page/close/) for the iteration to be able to finish.
```

Reply:
Fixed in [d49b29c](https://github.com/grafana/k6-docs/pull/1308/commits/d49b29cc34257f3a4e508c4449b308530eb455a8). Thanks for your review :+1: